packages/docs/theme.config.jsx

```diff
@@ -34,7 +34,7 @@ export default {
     return <OramaLogo isDark={isDark} />
   },
   chat: {
-    link: 'https://join.slack.com/t/orama-community/shared_invite/zt-1gzvj0mmt-yJhJ6pnrSGuwqPmPx9uO5Q',
+    link: ' https://orama.to/slack',
```

Comment:
please avoid space here before the link

Reply:
Yes sorry, I've seen it after the push. Will delete it asap

Reply:
Yeah I fear it's just linting errors :)